app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command[0:2] != "cp" or command.count(" ") != 2:
```

Comment:
The check `command[0:2] != "cp"` should be `command.split()[0] != "cp"` to ensure the command starts with 'cp' as a separate word.